package.json

```diff
@@ -36,12 +37,11 @@
     "@types/react": "^18.2.7",
     "@types/react-dom": "^18.2.4",
     "@types/sharp": "^0.31.0",
-    "@typescript-eslint/eslint-plugin": "^7.1.0",
-    "@typescript-eslint/parser": "^7.1.0",
```

Comment:
I had to import `typescript-eslint` to get at the recommended configs in the modern flat format, and these packages are part of `typescript-eslint`